frontend/src/pages/requests/StatusToolTip.tsx

```diff
@@ -0,0 +1,59 @@
+import { ITooltipParams } from "ag-grid-community";
+import { AgGridReact } from "ag-grid-react";
+import { useState } from "react";
+
+export const StatusTooltip = (props: ITooltipParams) => {
+  const data = props.api!.getDisplayedRowAtIndex(props.rowIndex!)!.data;
+
+  const [columnDefs] = useState([
+    { field: "fieldName", headerName: "Field Name" },
+    {
+      field: "report",
+      headerName: "Report",
+      wrapText: true,
+      autoHeight: true,
+      flex: 2,
+    },
+  ]);
+
+  let validationReportMap = new Map();
+  try {
+    validationReportMap = new Map(
+      Object.entries(JSON.parse(data?.hasStatusStatuses[0].validationReport))
+    );
+  } catch (e) {
+    let cleanedReport = (data?.hasStatusStatuses[0].validationReport)
+      .replace("{", "")
+      .replace("}", "");
+    let reportArray = cleanedReport.split(",");
+    for (var r of reportArray) {
+      let splitReport = r.split("=");
+      validationReportMap.set(splitReport[0], splitReport[1]);
```

Comment:
If the input is `"{baitSet=missing, specimenType=missing}"`, this code would produce 
`[    { fieldName: 'baitSet', report: 'missing' },    { fieldName: ' specimenType', report: 'missing' }  ]` (Note the extra space before specimenType).

I think this could be fixed with via `trim()`.
`validationReportMap.set(splitReport[0].trim(), splitReport[1]);`